tests/test_diffraction_DiffractionPattern.py

```diff
@@ -88,68 +88,75 @@ def test_attribute_shapes(self):
         assert dp.to_image().shape == (output_size, output_size, 4)
         assert dp.N_points == len(positions)
 
-    def test_center_unordered(self):
+    vals = (
+        "grid_size, output_size, view_orientation",
+        [
+            (grid_size, output_size, view_orientation)
+            for grid_size in (255, 256)
+            for output_size in (255, 256)
+            for view_orientation in rowan.random.rand(10)
+        ],
+    )
+
+    @pytest.mark.parametrize(*vals)
```

Comment:
`vals` is only used a couple of times. I'd rather copy-paste the parameterizations and keep them close to the test definitions.

```suggestion
    @pytest.mark.parametrize("grid_size", (255, 256))
    @pytest.mark.parametrize("output_size", (255, 256))
    @pytest.mark.parametrize("view_orientation", rowan.random.rand(10))
```

Reply:
Better yet, we could define `test_center_unordered` and ``test_center_ordered` together by parametrizing the system.
```python
    @pytest.mark.parametrize("box, positions", (
        freud.data.make_random_system(box_size=10, num_points=1000),
        freud.data.UnitCell.bcc().generate_system(10),
    ))